package.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
I can take care of bumping the addon version number when I release. It's best to leave this value unchanged for now, just in case I have to squeeze in another release before this lands.